personalpalate/meal_rec.py

```diff
@@ -0,0 +1,71 @@
+import math
+import statistics
+from collections import Counter
+from datetime import date, timedelta, datetime
+from personalpalate.orm.model import Meal
+
+
+'''
+# Function to construct the probability mass function for the dataset
+# Parameters
+#   - meals: list of meals in a user's dataset
+#   - past_choices: list containing each meal chosen and the date chosen
+# Returns: recommended meal as a string
+'''
+def construct_pmf(meals, past_choices):
+
+    # frequency of each mealName in the dataset
+    freq = Counter(meals.mealName)
+    total_count = len(meals)
+
+    # Constructing the PMF based on the frequency in the dataset
+    pmf = {value: freq[values] / total_count for values in freq}
+
+    seasonal_weight = 0.0   # positive seasonal weight
```

Comment:
Move these two variables to just being declared in their respective loops, since thats the only place they're used. AKA, remove these two variables here